core/dbt/cli/main.py

```diff
@@ -6,7 +6,10 @@
 from dbt.adapters.factory import adapter_management
 from dbt.cli import params as p
 from dbt.cli.flags import Flags
+from dbt.events.functions import setup_event_logger
 from dbt.profiler import profiler
+import dbt.tracking
```

Comment:
Let's use the [google styleguide for importing](https://google.github.io/styleguide/pyguide.html#22-imports) and import specific things from the module instead of the whole module.
`from dbt.tracking import initalize_from_flags, track_run`